cpp/test/Slice/errorDetection/EnumeratorValues.err

```diff
@@ -1,5 +1,5 @@
 EnumeratorValues.ice:12: value for enumerator `C' is out of range
-EnumeratorValues.ice:12: enumerator `C' has the same value as enumerator `B'
 EnumeratorValues.ice:13: value for enumerator `D' is out of range
 EnumeratorValues.ice:14: value for enumerator `E' is out of range
 EnumeratorValues.ice:15: value for enumerator `F' is out of range
+EnumeratorValues.ice:15: enumerator `F' has the same value as enumerator `B'
```

Comment:
My changes have a small impact on the ordering/logic we use to check for duplicate enumerators,
hence this small change in the error detection test.